client/apps/city_issues/views.py

```diff
@@ -177,6 +179,13 @@ def get_queryset(self):
         """Adds sorting"""
         queryset = super(CheckIssues, self).get_queryset()
         order_by = self.request.GET.get('order_by', 'title')
```

Comment:
there better to use cleaned_data
self.form.cleaned_data['order_by']

Reply:
In this case I can't do so.

Reply:
In this case I can't do so